static/js/components/HeaderContent.jsx.template

```diff
@@ -5,19 +5,17 @@ import Box from "@material-ui/core/Box";
 import Responsive from "./Responsive";
 import ProfileDropdown from "./ProfileDropdown";
 import Logo from "./Logo";
-import QuickSearchBar from "./QuickSearchBar";
 
 import styles from "./Main.css";
 
 const HeaderContent = () => (
   <div className={styles.topBannerContent}>
-    <div style= {%- raw -%} {{display: "inline-flx", flexDirection: "row" }} {%- endraw -%} >
+    <div style= {%- raw -%} {{display: "inline-flex", flexDirection: "row" }} {%- endraw -%} >
```

Comment:
No longer necessary?

Reply:
It is -- need to spell `flex` whrite!

Reply:
I meant, is that whole line still needed :)